notifier/github/client.go

```diff
@@ -69,10 +69,11 @@ type service struct {
 // NewClient returns Client initialized with Config
 func NewClient(cfg Config) (*Client, error) {
 	token := cfg.Token
-	token = strings.TrimPrefix(token, "$")
-	if token == EnvToken {
-		token = os.Getenv(EnvToken)
```

Comment:
Since now the `EnvToken` is used only in testing, we can remove the variable.

Reply:
Yeah, true. Let me remove it 👍

Reply:
Thank you for your review, removed → https://github.com/mercari/tfnotify/pull/112/commits/e5e8b67178b2286088e63551c7bfa35b9d30006c